templates/ci/php.yml.metadata

```diff
@@ -0,0 +1,3 @@
+{
+  "iconTaskName": ""
```

Comment:
The template uses `ArchiveFiles`, but we probably want to a web-oriented icon or the php logo.